WalletWasabi.Tests/RegressionTests/Common.cs

```diff
@@ -84,8 +84,8 @@ private static async Task AssertFiltersInitializedAsync(RegTestFixture regTestFi
 			var serviceConfiguration = new ServiceConfiguration(MixUntilAnonymitySet.PrivacyLevelSome.ToString(), 2, 21, 50, regTestFixture.BackendRegTestNode.P2pEndPoint, Money.Coins(WalletWasabi.Helpers.Constants.DefaultDustThreshold));
 
 			var dir = Helpers.Common.GetWorkDir(callerFilePath, callerMemberName);
-			var indexStore = new IndexStore(Path.Combine(dir, "indexStore"), network, new SmartHeaderChain());
-			var transactionStore = new AllTransactionStore(Path.Combine(dir, "transactionStore"), network);
+			await using var indexStore = new IndexStore(Path.Combine(dir, "indexStore"), network, new SmartHeaderChain());
```

Comment:
This is IMO not correct because you want to use `indexStore` outside of `InitializeTestEnvironmentAsync`.